.github/workflows/build.yml

```
@@ -226,12 +222,21 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Install process user data file script dependencies
+        run: python -m pip install boto3 jinja2
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the Windows admin password
+        # pulled from AWS Parameter Store. Return the admin password to
+        # pass into Packer via command line variable.
```

Wanted to clean up this comment a bit to better represent what the script is doing.

```suggestion
        # Process the user data script to set the Windows Administrator
        # password to the value retrieved from AWS Parameter Store. Set a step
        # output to use the retrieved password as the winrm_password value
        # with Packer via command line argument.
```

---

@mcdonnnj resolved with [189a1b1](https://github.com/cisagov/windows-server-packer/pull/10/commits/189a1b18233e92d79007a097e60f10dbcca0e493)